frontend/packages/ux-editor/src/testing/mocks.tsx

```diff
@@ -58,7 +61,7 @@ export const appStateMock: IAppState = {
   widgets: null,
 };
 
-export const queriesMock: ServicesContextProps = {
+export const queriesMock: Partial<ServicesContextProps> = {
```

Comment:
Vi bør heller legge til mocks for funksjonene som mangler her så vi unngår typefeil og uriktige testresultater. `ServicesContextProvider` skal ta inn `ServicesContextProps`, ikke `Partial<ServicesContextProps>`. Litt snodig at det ikke feiler egentlig.
```suggestion
export const queriesMock: ServicesContextProps = {
```